syntax = "proto3";

package swarm;

import "simulation/swarm/proto/agent.proto";

// Swarm configuration.
message SwarmConfig {
  // Step time in seconds.
  float step_time = 1;

  // Missile swarm configurations.
  repeated AgentSwarmConfig missile_swarm_configs = 2;

  // Target swarm configurations.
  repeated AgentSwarmConfig target_swarm_configs = 3;
}
